### DISCUSSION TITLE
iOS and Android SDKs is not updated

### DISCUSSION THREAD

**Discussion by valentirob** [microsoft/AdaptiveCards#8450] [General]
Do you have plan to update iOS and Android SDKs ?

**Comment by jwoo-msft** [comment]
Hello @valentirob,

We are planning to roll out a new release in mid-May.
We will keep you informed about our plans for future updates beyond May.
In the meantime, we would greatly appreciate it if you could share some insights into how you and your team utilize AdaptiveCards in your projects.

Thank you!
